website/databend/mkdocs.yml

```diff
@@ -229,6 +229,7 @@ nav:
               - POSITION: sqlstatement/string-functions/position.md
               - INSERT: sqlstatement/string-functions/insert.md
               - FIELD: sqlstatement/string-functions/field.md
+              - ELT: sqlstatement/string-functions/elt.md
```

Comment:
Hi @mshauneu , with the #3210 merge we are gradually migrating to Docusaurus and the old MkDocs will be obsolete. You may need to sync with the main branch and reorganise this documentation.

Reply:
@PsiACE changed